src/components/Projects/WBS/WBSDetail/AddTask/AddTaskModal.jsx

```diff
@@ -486,7 +486,8 @@ function AddTaskModal(props) {
                       onChange={e => setHoursBest(e.target.value)}
                       onBlur={() => calHoursEstimate()}
                       id="bestCase"
-                      className="w-50 ml-2"
+                      className=""
```

Comment:
If the className attribute is not being used and you prefer inline styles, please remove the className attribute.